packages/components/file-uploader-input/src/file-uploader-input.js

```diff
@@ -0,0 +1,233 @@
+import {
+	TSElement,
+	unsafeCSS,
+	html,
+	customElementDefineHelper
+} from '@tradeshift/elements';
+import css from './file-uploader-input.css';
+
+customElementDefineHelper(
+	'ts-file-uploader-input',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				rtl: { type: Boolean, reflect: true },
+				disabled: { type: Boolean, reflect: true },
+				multiple: { type: Boolean, reflect: true },
+				size: { type: String, reflect: true },
+				acceptedFileExtensions: {
+					type: Array,
+					reflect: true,
+					attribute: 'accepted-file-extensions'
+				},
+				disableDragAndDrop: {
+					type: Boolean,
+					reflect: true,
+					attribute: 'disable-drag-and-drop'
+				},
+				fileUploadWrapper: {}
+			};
+		}
+
+		constructor() {
+			super();
+			this.multiple = false;
+			this.disableDragAndDrop = false;
+			this.size = 'full';
+			this.dragCounter = 0;
+			this.handleUpload = this.handleUpload.bind(this);
+			this.dragEnterHandler = this.dragEnterHandler.bind(this);
+			this.dragOverHandler = this.dragOverHandler.bind(this);
+			this.dragLeaveHandler = this.dragLeaveHandler.bind(this);
+			this.dropHandler = this.dropHandler.bind(this);
+		}
+
+		get dropBox() {
+			return html`
+				<div class="draggable-info drop">
+					<div class="icon-wrapper">
+						<div class="icon">
+							<svg
+								width="20px"
```

Comment:
We need change those to variables

Reply:
Yeah, all of those icons are temporary! One of the first new components will be the icon component which includes all of these icons in maintainable and customisable way